### PR TITLE
System containers: use SD_NOTIFY for openvswitch

### PR DESCRIPTION
**Pull request by giuseppe**
The most important change is that the openvswitch system container uses SD_NOTIFY to notify systemd when it is ready.  Plus some other minor fixes I've accumulated here.  Some of them are necessary for the cluster to come back online after a reboot.

### CODE REVIEW

**Comment by giuseppe** [comment]
[test]

File: images/node/system-container/tmpfiles.template [hunk 1]
```diff
@@ -1,4 +1,5 @@
-d   /var/lib/origin - - - - -
+d   $ORIGIN_DATA_DIR - - - - -
 d   /var/lib/cni - - - - -
 d   /var/run/secrets - - - - -
-d   /etc/origin/sdn - - - - -
+d   $ORIGIN_CONFIG_DIR/sdn - - - - -
+d   $ORIGIN_CONFIG_DIR/cloudprovider
```
Comment (rhatdan) [inline images/node/system-container/tmpfiles.template]: Missing RETURN?

**Comment by rhatdan** [comment]
Other then NIT, LGTM

File: images/node/system-container/tmpfiles.template (hunk 1 above)
Comment (giuseppe) [inline images/node/system-container/tmpfiles.template]: Thanks for the review, I have pushed a new patch to add the missing '- - - - -\n'

**Comment by giuseppe** [comment]
[merge]

**Comment by sdodson** [comment]
[merge]

**Pull request merged**